thunder/tests/opinfos.py

```diff
@@ -1811,6 +1811,30 @@ def hardshrink_opinfo_factory(lambds):
 elementwise_unary_ops.append(selu_opinfo)
 
 
+tanhshrink_opinfo = OpInfo(
+    ltorch.tanhshrink,
+    dtypes=(datatypes.inexact,),
+    sample_input_generator=elementwise_unary_generator,
+    torch_reference=torch.nn.functional.tanhshrink,
+    test_directives=(
+        # Torch doesn't support CPU float16 or complex32 tanhshrink
+        DecorateInfo(
+            pytest.mark.xfail,
+            "test_core_vs_torch_consistency",
+            dtypes=(datatypes.float16, datatypes.complex32),
+            devicetypes=(devices.DeviceType.CPU,),
+        ),
+        # NOTE Nvfuser fails with AssertionError: Tensor-likes are not close!
```

Comment:
Interesting! How far off are the results? Let's include the atol and rtol in the comment. Can we set a custom comparator to get the tests to pass?

Reply:
Sure, custom comparison added.